<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvestigateProcessWindow</class>
 <widget class="QWidget" name="InvestigateProcessWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Process Investigator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="AssetResources.qrc">
    <normaloff>:/assets/PeaceMakerIcon.ico</normaloff>:/assets/PeaceMakerIcon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #404040;</string>
  </property>
  <widget class="QTableWidget" name="ProcessInformationTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>440</width>
     <height>160</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>20</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="ProcessStackHistoryTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>440</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="BasicProcessInformationLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>179</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>&lt;img src=&quot;:/assets/BasicProcessInformation.png&quot;/&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="CreatorStackHistoryLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>148</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>&lt;img src=&quot;:/assets/CreatorStackHistory.png&quot;/&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ImagesLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>&lt;img src=&quot;:/assets/Images.png&quot;/&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ImageStackHistoryLabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>200</y>
     <width>138</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>&lt;img src=&quot;:/assets/ImageStackHistory.png&quot;/&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="ImagesTable">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>440</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="ImageStackHistoryTable">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>440</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="AssetResources.qrc"/>
 </resources>
 <connections/>
</ui>
